pppd/sys-linux.c

```diff
@@ -667,6 +668,19 @@ static int make_ppp_unit(void)
 {
 	int x, flags;
 
+	// Variable to create regex
+	regex_t *regex;
```

Comment:
This need not be a pointer, can be allocated on-stack (it's small enough in my opinion at 64 bytes).